node/alphanet/src/testing_utils.rs

```diff
@@ -122,3 +128,29 @@ where
         }
     }
 }
+
+/// Generates chainspec for running multiple nodes.
+pub fn generate_test_chain_spec(account_names: &Vec<String>, balance: u64) -> ChainSpec {
```

Comment:
remove then configure_chain_spec (idae was to add configuration there).
Also add option to specify number of authorities out of this account_names

Reply:
I think there is still usage of `configure_chain_spec`, because `generate_test_chain_spec` does not strictly supersede it. Also for the sake of unblocking everyone I'll just go ahead and merge this.

>Also add option to specify number of authorities out of this account_names
Not sure I understand what it means.

Reply:
You currently making all the accounts authorities
Instead it should be k out of n accounts are authorities, the rest are just regular accounts. E.g. k should be a parameter.